src/midcom/datamanager/extension/type/select.php

```diff
@@ -44,9 +44,18 @@ public function configureOptions(OptionsResolver $resolver)
             }
         };
 
+        $multiple_options = function (Options $options) {
+            $return_options = array();
+            if (isset($options['type_config']['allow_multiple'])) {
+                $return_options = $options['type_config']['allow_multiple'];
+            }
+            return $return_options;
+        };
```

Comment:
if `$options['type_config']['allow_multiple']` is not set, you return an array instead of a boolean. That should be fixed. 

In general, the whole function could be simplified to look like this:

```php
        $multiple_options = function (Options $options) {
            return !empty($options['type_config']['allow_multiple']));
        };
```